### DISCUSSION TITLE
Write a concise mission statement for the VTP Web Client project

### DISCUSSION THREAD

**Discussion by stratofax** [TrustTheVote-Project/VTP-web-client#3] [Ideas]
The VTP Web Client project needs a concise mission statement in the README.md file, so contributors and team members all share a clear understanding of the project's strategic goals.

> The README.md file should feature a concise, clear mission statement to help visitors quickly understand its purpose and decide their interest. This statement should be brief, specific, and positioned prominently. An effective mission statement succinctly describes the project's function, target audience, and key features, assuming a basic level of technical knowledge from the reader. This approach ensures that the content resonates with the intended audience, encouraging further exploration of the project.

-- Summarized from [Producing Open Source Software, Chapter 2. Getting Started: Mission Statement](https://producingoss.com/en/getting-started.html#mission-statement)

Let's use this discussion to draft a short mission statement for the README.

**Comment by windoverwater** [comment]
Question - there are 5 repos for VTP at the moment.  This one (VTP-web-client) is probably where the creatives (UX and UI designers) will land/have the most interest.  Backend developers/security folks will probable go to the main VoterTrackerPlus repo.

Thoughts on having just one such readme or a readme per the type of person or for type of repo content?

**Reply by stratofax** [reply]
@windoverwater I think it's best to have a repo-specific mission statement that's targeted at the exact audience for that repo. You've identified the different audiences, and their different interests, quite succinctly. The mission statement for this repo should reflect that insight.

Another way to look at this: if the API is well designed, as I suspect it is, the UI/UX folks won't have to worry about the security and backend development issues at all if they focus on this repo.

The README.md should contain some links to the other VTP repos, or at least the ones that interface directly with this one, and vice versa (so the other related repos link back to this one). That way, if someone who's potentially more interested in (say) security may find their way to the more appropriate project, even if they came across this repo first.
